ols/constants.py

```diff
@@ -207,3 +207,4 @@ class QueryValidationMethod(StrEnum):
 # default indentity for local testing and deployment
 DEFAULT_USER_UID = "c1143120-551e-4a47-ad47-2748d6f3c81c"
 DEFAULT_USER_NAME = "OLS"
+DEFAULT_KUBEADMIN_UID = "b6553200-0f7b-4c82-b1c5-9303ff18e5f0"
```

Comment:
please add a TODO here, we want to replace this value w/ something that's dynamically generated based on the cluster and then stays constant.  That way the kubeadmin user on a given cluster has a unique uuid relative to the kubeadmin user on any other cluster.